Makefile

```diff
@@ -233,7 +233,11 @@ test-python: pyenv az
 
 
 shared-cluster-login:
-	@oc login ${SHARED_CLUSTER_API} -u kubeadmin -p ${SHARED_CLUSTER_KUBEADMIN_PASSWORD}
+	RP_MODE="production" \
+	az aro get-admin-kubeconfig \
```

Comment:
If we want to preserve the existing behavior of this command, we could instead use `az aro list-credentials`, which outputs username/password as JSON, and then invoke `oc login` with those parameters. 

IMO I prefer to have a dedicated kubeconfig for shared cluster on my machine, so I like the current implementation more, though I'd suggest changing the file name to something like `shared-cluster.admin.kubeconfig` to not clobber any dev cluster's kubeconfigs retrieved via `make admin.kubeconfig`.